app/styles/app.scss

```diff
@@ -264,4 +264,50 @@ body {
    font-weight: bold;
 }
 
+.search-container{
+   padding-left:320px;
+ 
+}
+
+input.light {
+   width:400px;
+   height:29px;
+   border: 1px solid #555;
+   display: block;
+   padding: 9px 4px 9px 40px;
+   margin-top: 10px;
+   font-weight: 400;
+   font-size: 14px;
+   line-height: 18px;
+   color: #183247;
+   background-color: #fff;
+   background: transparent url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' class='bi bi-search' viewBox='0 0 16 16'%3E%3Cpath d='M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z'%3E%3C/path%3E%3C/svg%3E") no-repeat 13px center;
+ }
+
+ .nodata {
+  margin-left: 500px;
+  margin-top: 100px;
+ display: flex;
+ flex-direction:column;
+ justify-content: center;
+
+ }
+.nodataImage {
+   align-self: center;
+}
+
+
+ .nodatamessage1 {
+   font-weight: 600;
+   font-size: 16px;
+   padding-top:10px;
+   
+ }
+
+ .nodatamessage2 {
+   font-weight: 400;
+   font-size: 14px;
+   padding-top:5px;
+ }
```

Comment:
`nodatamessage1`, `nodatamessage2` are inside `nodataImage` class. So, we can maintain hierarchy in the styles.

Reply:
Done.